ui/app/templates/components/transform-role-edit.hbs

```diff
@@ -30,7 +30,7 @@
 {{#if (eq this.mode "show")}}
   <Toolbar>
     <ToolbarActions>
-      {{#if this.capabilities.canDelete}}
+      {{#if this.model.updatePath.canDelete}}
```

Comment:
Is there a reason we don't want to make this a computed on the model? I ask because our new pattern has been to explicitly check for false so we don't accidentally hide links while the return is `undefined`. I'll defer to you, just wanted to call out!

Reply:
I went that method at first, and hit some gnarly model isDestroyed during transitioning problems. I avoided that rabbit hole and stuck with this.

Reply:
Updated a comment [here.](https://github.com/hashicorp/vault/pull/28563/files#r1785379398)